README.md

```diff
@@ -14,16 +14,23 @@ Building for Fedora
 
 Packages needed for the build, or the build requires:
 
+* make
 * check-devel
 * cmake
 * expat-devel
 * gcc
 * libsolv-devel
 * libsolv-tools
+* librepo-devel
 * python-devel (or python3-devel for Python 3 build)
 * python-nose (or python3-nose for Python 3 build)
 * rpm-devel
 * zlib-devel
+* glib2-devel
+* gtk-doc
+* gobject-introspection-devel
+* redhat-rpm-config
```

Comment:
@jlebon It should `never` be explicitly required in a package because it's `always` present in the buildroot, so it shouldn't be mentioned as a hard requirement.

Besides, I maintain the DNF stack, including libhif, on Mageia, and I'd be furious if it explicitly required `redhat-rpm-config` because ours is `rpm-mageia-setup`.

In any case, this is a non-requirement. Take it out.